docs/implementation.adoc

```diff
@@ -357,6 +357,18 @@ public class AccurevSCMSource extends SCMSource {
                     new NonLocalizable("Streams")
                     // Better: Messages._AccurevSCMSource_StreamHeadCategory()
                 ),
+                new UncategorizedSCMSourceCategory(
+                    new NonLocalizable("Gated Streams")
+                    // Better: Messages._AccurevSCMSource_GatedStreamHeadCategory()
+                ),
+                new ChangeRequestSCMHeadCategory(
+                    new NonLocalizable("Staging Streams")
+                    // Better: Messages._AccurevSCMSource_StagingStreamHeadCategory()
+                )
+                new ChangeRequestSCMHeadCategory(
```

Comment:
I think workspaces deserve their own category... though I am unsure if you really want them.

OTOH it would be cool to enable a dev to get CI reports of their local changes

Reply:
That's my exact thought too, having it as a trait but not by default I think would be awesome 👍